services/meta/association.md

```diff
@@ -3,29 +3,40 @@ This service allows creating a direct association between a source device (e.g.
 
 This service intends to replace `group` commands in `dev_sys` service.
```

Comment:
Add a short description that associations are meant to link a controlling device with another controlled device through some basic behaviors.

Reply:
There's a description above already, I added info that they need to be controllable